corehq/apps/users/role_utils.py

```diff
@@ -55,6 +55,4 @@ def init_domain_with_presets(domain):
 
 
 def get_all_role_names_for_domain(domain_name):
-    presets = set(UserRolePresets.NAME_ID_MAP.keys())
-    custom = set([role.name for role in UserRole.by_domain(domain_name)])
-    return presets | custom
+    return set([role.name for role in UserRole.by_domain(domain_name)])
```

Comment:
can you share more on why this is okay to do? exclude presets.

Reply:
Because all the roles are created in the DB when the domain is created.